lib/smart_answer_flows/coronavirus-find-support/questions/nation.erb

```diff
@@ -1,3 +1,7 @@
+<% text_for :caption do %>
+  &nbsp;
+<% end %>
```

Comment:
Hi @theseanything - for this page and `need_help_with` there is no group, other text we can use and the flow name is not shown.  Do you know a better way that `&nbsp;`?  Again, it works - but feels hacky.

Reply:
Wouldn't it be better to add a default behaviour for `caption` so that when not given in the erb, it inserts a non-breaking space.

Reply:
Looking again. This is the method that needs updates:

_app/presenters/question_presenter.rb_
```ruby
def caption
  return @renderer.content_for(:caption) if @renderer.content_for(:caption).present?
 
  @flow_presenter.title
end
```
I think by passing in the non-breaking space, it forcing the rendering of the content_for(:caption) which then renders that space.

Reply:
Have decided to go with the default flow title for now.